### PR TITLE
Watermark on Image 

### PR DESCRIPTION
**Pull request by aishwaryachand**
# Description

Watermark on image 

## Fixes: #546 
 issue 
Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Watermark On Image/Watermark_on_image.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#Import required Image library
+from PIL import Image, ImageDraw, ImageFont
+
+#Create an Image Object from an Image
+im = Image.open('test_image.jpg')
+width, height = im.size
+
```
Comment (santushtisharma10) [inline Watermark On Image/Watermark_on_image.py:7]: @aishwaryachand take the image input from the user.

File: Watermark On Image/Watermark_on_image.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+draw = ImageDraw.Draw(im)
+text = "Amazing Python Scripts"
+
```
Comment (santushtisharma10) [inline Watermark On Image/Watermark_on_image.py:10]: @aishwaryachand  take the text input from the user

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]

File: Watermark On Image/Watermark_on_image.py (hunk 2 above)
Comment (aishwaryachand) [reply Watermark On Image/Watermark_on_image.py]: changes made ,pls check

File: Watermark On Image/Watermark_on_image.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#Import required Image library
+from PIL import Image, ImageDraw, ImageFont
+
+
+path = input("Input the path of the image: ")
+path = path.strip('""')
```
Comment (antrikshmisri) [inline Watermark On Image/Watermark_on_image.py:6]: This line is not needed , you can remove it

**Review by antrikshmisri — CHANGES_REQUESTED** [review]

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@aishwaryachand the text goes out of the image if the text is longer. Use textwrap to solve it.

File: Watermark On Image/Watermark_on_image.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+width, height = im.size
+
+draw = ImageDraw.Draw(im)
+
+#input the text that needs to be printed
+text = input('Enter the text for the watermark: ' )
+font = ImageFont.truetype('arial.ttf', 60 )
+textwidth, textheight = draw.textsize(text, font )
+
+# calculate the x,y coordinates of the text
```
Comment (kaustubhgupta) [inline Watermark On Image/Watermark_on_image.py:17]: Include the options to choose the position of the watermark: Top Right, Top Left, Bottom Right, Bottom Left, and Center.

File: Watermark On Image/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+Using this code , we will be able to create a watermark on a given image. According to the value assigned to the margin ,the position of the watermark can be moved.
+
+## Dependencies
+We will be using the PIL -Python Imaging Library for this program.
+
+## Sample Output 
+
+ORIGINAL IMAGE
+
+![plot1](./test_image.jpg)
```
Comment (kaustubhgupta) [inline Watermark On Image/README.md:12]: Images not rendered.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@aishwaryachand Updates on this?

**Comment by kaustubhgupta** [comment]
@aishwaryachand final updates?

**Comment by aishwaryachand** [comment]
> @aishwaryachand final updates
Almast done... Will make the pr today .. Sry for taking so long

**Comment by kaustubhgupta** [comment]
@aishwaryachand you need to update the code today only else this issue will be assigned to someone else

**Comment by kaustubhgupta** [comment]
@aishwaryachand as informed earlier also, this issue will now be assigned to someone else and this PR closed

**Comment by aishwaryachand** [comment]
> @aishwaryachand as informed earlier also, this issue will now be assigned to someone else and this PR closed

 I wasn't able to solve one of the issue that you guys had earlier pointed so I didn't update the code.

**Comment by kaustubhgupta** [comment]
> > @aishwaryachand as informed earlier also, this issue will now be assigned to someone else and this PR closed
> 
> I wasn't able to solve one of the issue that you guys had earlier pointed so I didn't update the code.

You need to inform us so that we could suggest some alternative/ our own implementation or something else. But not responding is not a solution

**Pull request closed without merging**